packages/pharaoh/lib/src/http/response.dart

```diff
@@ -1,6 +1,7 @@
 import 'dart:convert';
 import 'dart:io';
 import 'package:http_parser/http_parser.dart';
+import 'package:pharaoh/src/utils/utils.dart';
```

Comment:
I'll prefer we use relative import style. `import '../utils/utils.dart';`

Reply:
done